src/test/java/tests/testng/JsonParserTest.java

```diff
@@ -70,25 +59,12 @@ public void filesQuantity() {
 
     @Parameters({"cart0"})
     @Test(groups = "goodTests", testName = "Saving of Cart with RealItems and VirtualItems")
-    public void checkCartItemsSaving(String cart0){
-        Cart cart = parser.readFromFile(new File("src/main/resources/" + FILE_NAME_LIST.get(Integer.valueOf(cart0)) + ".json"));
-        Field realItems = null;
-        Field virtualItems = null;
-        try {
-            realItems = Cart.class.getDeclaredField("realItems");
-            virtualItems = Cart.class.getDeclaredField("virtualItems");
-            realItems.setAccessible(true);
-            virtualItems.setAccessible(true);
-            List<RealItem> realItemList = (List<RealItem>) realItems.get(cart);
-            List<RealItem> vitalItemList = (List<RealItem>) virtualItems.get(cart);
-
-            Assertions.assertTrue(
-                    CollectionUtils.isNotEmpty(realItemList) && CollectionUtils.isNotEmpty(vitalItemList)
-                            && realItemList.size() == 1 && vitalItemList.size() == 1
-            );
-        } catch (NoSuchFieldException | IllegalAccessException ex) {
-            throw new NoSuchFileException("Error to read the file", ex);
-        }
+    public void checkCartItemsSaving(int cart0){
+        Cart cart = parser.readFromFile(new File("src/main/resources/" + FILE_NAME_LIST.get(cart0) + ".json"));
+        Assertions.assertAll("Cart name and total price are correct",
```

Comment:
please do not use junit assertions in testNg test, try to find proper one in testNg

Reply:
fixed

Reply:
fixed